suite-native/app/android/fastlane/Fastfile

```diff
@@ -79,6 +79,17 @@ platform :android do
       skip_upload_apk: true
     )
 
+    package_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name)
+    sentry_upload_sourcemap(
+      auth_token: ENV["SENTRY_AUTH_TOKEN"],
+      org_slug: 'satoshilabs',
+      project_slug: 'suite-native',
+      version: ENV["VERSION"],
+      app_identifier: package_name, 
+      build: ENV["BUILD_NUMBER"]
+      sourcemap: ['index.android.bundle', 'index.android.bundle.map'],
+      rewrite: true
```

Comment:
what does this parameter do?

Reply:
Hard to say from the code. I would say it rewrites the sourcemap for the version and build. Even though we bump the build number, I suggest to keep it there just for sure.